js/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "wrongsecrets",
+  "version": "1.3.1",
+  "description": "Examples with how to not use secrets",
+  "main": "index.js",
```

Comment:
when running the app from IntelliJ i get 
<img width="424" alt="image" src="https://user-images.githubusercontent.com/1457214/159909376-c7f6a458-2ca5-4ae6-93ed-18857c58d8a3.png">
can you help me a bit :) ?

Reply:
In the above comment you had problem with `mvn clean install`. It is required to build JS resource file for Spring Boot. I think it can be the reason.

Reply:
mvn clean install did not crash, but still I get a 404...

Reply:
Should be fixed right now

Reply:
it is :D after clean& package.thanks!